app/src/components/app.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import css from '../../styles/index.scss';
 
-const ACCESS_TOKEN_REGEX = /#access_token=([a-zA-Z0-9.\-\_]*)(&[a-z=])?/g;
+const ACCESS_TOKEN_REGEX = /#access_token=([a-zA-Z0-9.\-_]*)(&[a-z=])?/g;
```

Comment:
can we triple test this please? just to be safe, we all had """fun""" times debugging regexes